app/src/main/kotlin/org/fossify/clock/helpers/DataExporter.kt

```diff
@@ -0,0 +1,51 @@
+package org.fossify.clock.helpers
+
+import org.fossify.clock.interfaces.JSONConvertible
+import org.fossify.commons.helpers.ExportResult
+import org.json.JSONArray
+import org.json.JSONObject
+import java.io.OutputStream
+
+object DataExporter {
+
+    fun exportData(
+        alarms: List<JSONConvertible>,
+        timers: List<JSONConvertible>,
+        outputStream: OutputStream?,
+        callback: (result: ExportResult) -> Unit,
+    ) {
+        if (outputStream == null) {
+            callback.invoke(ExportResult.EXPORT_FAIL)
+            return
+        }
+
+        val alarmsJsonArray = toJsonArray(alarms)
+        val timersJsonArray = toJsonArray(timers)
+
+        val jsonObject = JSONObject().apply {
+            put("alarms", alarmsJsonArray)
+            put("timers", timersJsonArray)
+        }
```

Comment:
After https://github.com/FossifyOrg/Clock/pull/53#discussion_r1936782984, we can use a data class for this as well.

```kotlin

 @kotlinx.serialization.Serializable
 data class AlarmsAndTimers(val alarms: List<Alarm>, val timers: List<Timer>)
```